seismic-app/src/components/Footer.js

```diff
@@ -0,0 +1,105 @@
+import "../css/footer.css";
+import React from "react";
+import { Row, Col, Typography, Input, Anchor } from "antd";
+
+const { Title, Text } = Typography;
+const { Link } = Anchor;
+
+function subscribe(event) {
+  const email = event.target.value;
+  console.log(`TODO: send email to ${email}`);
+}
+
+const text = {
+  color: "white",
+  textAlign: window.innerWidth > 768 ? "start" : "center"
+};
+
+const Footer = props => {
+  const [width, setWidth] = React.useState(window.innerWidth);
+  const [textStyle, setTextStyle] = React.useState({
+    textAlign: window.innerWidth > 768 ? "start" : "center"
+  });
+  const updateWidth = () => {
+    setWidth(window.innerWidth);
+    if (width < 768) {
+      textStyle.textAlign = "center";
```

Comment:
Not really sure about this one, but shouldn't this be done through `setTextStyle`, as it is state and not a simple var?